playground/frontend/lib/pages/embedded_playground/widgets/embedded_actions.dart

```diff
@@ -46,8 +46,9 @@ class EmbeddedActions extends StatelessWidget {
         Consumer<PlaygroundController>(
           builder: (_, controller, __) {
             final selectedExample = controller.selectedExample;
-            final hasGithubLink = selectedExample?.urlVcs != null;
```

Comment:
Too many `?` and `!`. Make a cut-off if `selectedExample == null`?

Reply:
done